### ISSUE TITLE
Design Revamp

### ISSUE THREAD

**Issue by SpandanMozumder** [suryanshsingh2001/MyLibrary#49]
I want to completely change the design and color combination of the website and add animations to the website to make it seem cleaner and more professional. Possible places to apply these changes are as follows - 

1. Make color and font changes to make the website look more antique.
2. The hover and card styling to be improved.
3. Card display and data display animations to be added.

**Comment by suryanshsingh2001** [comment]
@SpandanMozumder  Absolutely, you can work on this design revamp! Here’s a breakdown of tasks to help you implement the changes:

1. **Color and Font Overhaul:**
   - [ ] Select an antique color palette (earthy tones, muted colors).
   - [ ] Choose appropriate fonts (serif fonts or fonts with a vintage feel).
   - [ ] Apply the new colors and fonts across the website.

2. **Hover and Card Styling Improvements:**
   - [ ] Redesign hover effects for a smoother transition (e.g., fade-in, shadow).
   - [ ] Update card styles with new borders, backgrounds, and padding for a vintage look.
   - [ ] Ensure card spacing is adjusted for better readability and presentation.

3. **Animations for Card and Data Display:**
   - [ ] Add entrance animations for cards (e.g., fade-in, slide-in).
   - [ ] Implement hover-triggered micro-animations (e.g., zoom, slight rotations).
   - [ ] Use `react-spring` or `Framer Motion` for smooth, professional animations.

4. **Responsiveness:**
   - [ ] Test and adjust the new design elements for responsiveness on various screen sizes.
   - [ ] Ensure that hover effects and animations perform well on mobile devices.

5. **Performance Optimization:**
   - [ ] Optimize animations to minimize reflows and repainting.
   - [ ] Use GPU-accelerated animations where possible for better performance.
   - [ ] Ensure the website remains lightweight and fast after changes.
   
6. **UI Component Library Integration (Optional):**
   - [ ] Feel free to integrate UI component libraries like **shadcn**, **Chakra UI**, or any other that suits the design and functionality.
   - [ ] Ensure consistency with the new design theme if using a library.

**Comment by HoangPhiHong1** [comment]
Hello @suryanshsingh2001, can I work on these tasks as well? I have some experience working with UI stuffs.

**Comment by suryanshsingh2001** [comment]
> Hello @suryanshsingh2001, can I work on these tasks as well? I have some experience working with UI stuffs.

Absolutely, please feel free to collborate with @SpandanMozumder

**Comment by suryanshsingh2001** [comment]
### Gentle Reminder ⏰

Hi! When you're assigned an issue, please note that if there's **no activity for over 7 days**, we may need to _unassign it_ to ensure project progress. This helps keep things moving smoothly.

Thank you for your understanding! 🙏

**Comment by suryanshsingh2001** [comment]
@HoangPhiHong1 @SpandanMozumder , Please start speeding up the process, you will be unassigned by this week, due to inactivity.
